playground/ssr/server.js

```diff
@@ -57,6 +57,7 @@ export async function createServer(
       res.status(200).set({ 'Content-Type': 'text/html' }).end(html)
     } catch (e) {
       vite && vite.ssrFixStacktrace(e)
+      if (isTest) throw e
```

Comment:
My test could be flaky as I haven't figured out some case where there's actually unexpected but harmless errors. I'm ok with leaving this out in case it makes our CI unstable 🤔

Reply:
CI looks good for now. If we see it starts to bring issues we can remove it later.